sympy/core/power.py

```diff
@@ -371,7 +371,8 @@ def _n2(e):
                         if b.is_negative is True:
                             return S.NegativeOne**other*Pow(-b, e*other)
                         if b.is_extended_real is False:
-                            return Pow(b.conjugate()/Abs(b)**2, other)
+                            if b.conjugate() != Abs(b)**2/b:
+                                return Pow(b.conjugate()/Abs(b)**2, other)
```

Comment:
I wonder what this is for. This `b.conjugate()/Abs(b)**2` is generally just `1/b` but we only get here if we are computing `(1/b)**other`.

Reply:
I noted the place where it was actually tested previously in the notes. With this case:

```
>>> i = symbols('i', imaginary=True)
>>> sqrt(3/i)
sqrt(3)*(sqrt(-i)/Abs(i))
```

Without:

```
>>> sqrt(3/i)
sqrt(3)/sqrt(i)
```

See commit a6f583765039dda2c9181773a1dd9318448fd7ba.

Reply:
I'm wondering whether it is better just to remove this case

Reply:
@smichr Do you remember anything about this? Is there an issue it addressed?